templates/pulp.conf.erb

```diff
@@ -0,0 +1,59 @@
+#
+# WARNING: THIS CONFIGURATION WAS GENERATED BY KATELLO-CONFIGURE TOOL,
+# CHANGES WILL LIKELY BE OVERWRITTEN.
+#
+
+# Apache configuration file for pulp web services and repositories
+
+AddType application/x-pkcs7-crl .crl
+AddType application/x-x509-ca-cert .crt
+
+# Example ssl cert and key files to get you started.
+# This MUST match /etc/pulp/pulp.conf [security] 'cacert'.
+SSLCACertificateFile <%= scope.lookupvar("::pulp::ca_cert") %>
+
+# allow older yum clients to connect, see bz 647828
+SSLInsecureRenegotiation on
+
+WSGIProcessGroup pulp
+WSGIApplicationGroup pulp
+WSGIDaemonProcess pulp user=apache group=apache processes=3 display-name=%{GROUP}
+
+# DEBUG - uncomment the next 2 lines to enable debugging
+#WSGIRestrictStdin Off
+#WSGIRestrictStdout Off
+
+WSGISocketPrefix run/wsgi
+WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
+WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
+
+<Directory /usr/share/pulp/wsgi>
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
```

Comment:
For this you need to pull in the variable in config.pp like https://github.com/Katello/puppet-capsule/blob/8d221b918730cb312073b2e7c85a36e2852d4252/manifests/init.pp#L137-L138 does.